.github/workflows/ci.yaml

```diff
@@ -18,15 +18,17 @@ jobs:
       STRAPI_TOKEN: ${{ secrets.STRAPI_TOKEN }}
     steps:
       - uses: actions/checkout@v3
+
+      - name: Enable Corepack
+        shell: bash
+        run: corepack enable
+
       - uses: actions/setup-node@v4
         with:
           node-version: 22.14.0
-      - uses: pnpm/action-setup@v4.0.0
-        with:
-          version: 10.8.1
 
       - name: Install dependencies
-        run: pnpm install
```

Comment:
Attention si le ignore-scripts est utilisé légitimement par d'autres deps, ça peut poser problème ;

Aussi, conseil de mettre :
```
permissions:
  contents: read
  pull-requests: read
```
au tout début du workflow (https://docs.github.com/en/actions/tutorials/authenticate-with-github_token#modifying-the-permissions-for-the-github_token)